README.md

```diff
@@ -3,129 +3,123 @@
+* [Paste files/text and choose the recipient afterwards ](https://github.com/RobinLinus/snapdrop/pull/534)
 * [Prevent devices from sleeping on file transfer](https://github.com/RobinLinus/snapdrop/pull/413)
 * Warn user before PairDrop is closed on file transfer
 * Open PairDrop on multiple tabs simultaneously (Thanks [@willstott101](https://github.com/willstott101))
-* [Video and Audio preview](https://github.com/RobinLinus/snapdrop/pull/455) (Thanks [@victorwads](https://github.com/victorwads))
-* Switch theme back to auto/system after darkmode or lightmode is enabled
+* [Video and audio preview](https://github.com/RobinLinus/snapdrop/pull/455) (Thanks [@victorwads](https://github.com/victorwads))
+* Switch theme back to auto/system after dark or light mode is on
 * Node-only implementation (Thanks [@Bellisario](https://github.com/Bellisario))
-* Automatic restart on error (Thanks [@KaKi87](https://github.com/KaKi87))
+* Auto-restart on error (Thanks [@KaKi87](https://github.com/KaKi87))
 * Lots of stability fixes (Thanks [@MWY001](https://github.com/MWY001) [@skiby7](https://github.com/skiby7) and [@willstott101](https://github.com/willstott101))
 * To host PairDrop on your local network (e.g. on Raspberry Pi): [All peers connected with private IPs are discoverable by each other](https://github.com/RobinLinus/snapdrop/pull/558)
-* When hosting PairDrop yourself you can [set your own STUN/TURN servers](docs/host-your-own.md#specify-stunturn-servers)
-* Built-in translations
+* When hosting PairDrop yourself, you can [set your own STUN/TURN servers](docs/host-your-own.md#specify-stunturn-servers)
+* Translations.
 
 </details>
 
 ## Screenshots
 <img src="docs/pairdrop_screenshot_mobile.gif" style="max-height: 50vh">
 
-## PairDrop is built with the following awesome technologies:
-* Vanilla HTML5 / ES6 / CSS3 frontend
-* [WebRTC](http://webrtc.org/) / [WebSockets](http://www.websocket.org/)
-* [NodeJS](https://nodejs.org/en/) backend
-* [Progressive Web App](https://wikipedia.org/wiki/Progressive_Web_App)
+## Built with the following awesome technologies:
+* Vanilla HTML5 / JS ES6 / CSS 3 frontend
+* [WebRTC](http://webrtc.org/) / WebSockets
+* [Node.js](https://nodejs.org/en/) backend
+* [Progressive web app](https://wikipedia.org/wiki/Progressive_Web_App)
 * [IndexedDB API](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API)
 * [zip.js](https://gildas-lormeau.github.io/zip.js/)
-* [cyrb53](https://github.com/bryc) super fast hash function
-* [Weblate](https://weblate.org/) Web based localization tool
-
-Have any questions? Read our [FAQ](docs/faq.md).
+* [cyrb53](https://github.com/bryc/code/blob/master/jshash/experimental/cyrb53.js) super-fast hash function
+* [Weblate](https://weblate.org/) web-based localization tool
 
-You can [host your own instance with Docker](docs/host-your-own.md).
+[FAQ](docs/faq.md)
 
+[Host your own instance with Docker](docs/host-your-own.md).
```

Comment:
Much clearer, thanks! Maybe we could do "Host your own instance with Docker or Node.js"